util/fpga/bitstream_bisect.py

```diff
@@ -0,0 +1,406 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Efficiently bisect FPGA regressions with the bitstream cache.
+
+This tool bisects while minimizing the number of full bitstream builds. Under
```

Comment:
Maybe: calls to the slow command (It's also possible for tests that effect FPGAs and verilator to use verilator as the slow command)